### PR TITLE
Hungarian translation

### PR DESCRIPTION
**Pull request by sk-skuba**

### CODE REVIEW

File: app/src/main/res/values-hu/strings.xml [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+    <string name="cmd_timer_title">időzítő [hossz] [üzenet]</string>
+    <string name="cmd_timer_help">Egy visszaszámláló időzítő létrehozására használható az eszközön lévő alapértelmezett alkalmazás használatával. Arg nélküli használata az időzítő alkalmazás indításához szükséges. A hossznak másodpercben kell lennie, és 1-től 86400-ig (24 óra) terjedhet. Opcionálisan követheti egy üzenet sztring a visszaszámlálóban való megjelenítéshez.\nPéldák:\n\'időzítő 60\' beállítja az időzítőt 1 percre\n\'időzítő 3600 Vigyél ki a szemetet!\' beállítja az időzítőt 1 órára, egy megjelenítendő üzenettel.</string>
+    <string name="no_length_provided_launching_timer_app">Nincs megadva hossz. Az időzítő alkalmazás indítása.</string>
+    <string name="parsing_length_string">:: Hossz sztring értelmezése...</string>
+    <string name="invalid_length_string_provided">Érvénytelen hossz sztring megadva. Egy számot kell megadnia 1 és 86400 között.</string>
+    <string name="setting_timer">
+    <![CDATA[=> Időzítő beállítása...]]></string>
```
Comment (coderGtm) [inline app/src/main/res/values-hu/strings.xml:435]: Please edit this line to keep it in a single line

**Review by coderGtm — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-hu/strings.xml (hunk 1 above)
Comment (sk-skuba) [reply app/src/main/res/values-hu/strings.xml]: Edited :)

**Comment by coderGtm** [comment]
Thanks a lot for submitting Hungarian translations. Your efforts will make the Launcher available to Hungarians in their native language. Do you agree that the translations are accurate to the best of your knowledge?

**Comment by sk-skuba** [comment]
> Thanks a lot for submitting Hungarian translations. Your efforts will make the Launcher available to Hungarians in their native language. Do you agree that the translations are accurate to the best of your knowledge?

Thank You

**Review by coderGtm — APPROVED** [review]

**Pull request merged**